zeus/run/dataloader.py

```diff
@@ -523,12 +713,14 @@ def _end_prof(self, cutoff: bool) -> None:
 
         # Nothing left to do if the window was terminated prematurely.
         if cutoff:
```

Comment:
Maybe no need to do the cutoff detection if we decide to scale the profile window when it exceeds one epoch. Will test it!

Reply:
Yeah I also think that all cutoff-related things can be safely removed.